CNAME

```diff
@@ -1 +1 @@
-www.featureops.ai
\ No newline at end of file
+docs.hopsworks.ai
```

Comment:
just leave it like it is, and then github should fix itself when you change the setting.

Reply:
reverted